terraform/lambdas.tf

```diff
@@ -224,3 +224,17 @@ module "trigger_post_to_slack_esg_not_terminating" {
   lambda_function_arn  = "${module.lambda_post_to_slack.arn}"
   sns_trigger_arn      = "${module.ec2_instance_terminating_for_too_long_alarm.arn}"
 }
+
+module "trigger_post_to_slack_server_error_alb" {
+  source               = "./lambda/trigger_sns"
+  lambda_function_name = "${module.lambda_post_to_slack.function_name}"
+  lambda_function_arn  = "${module.lambda_post_to_slack.arn}"
+  sns_trigger_arn      = "${module.alb_server_error_alarm.arn}"
+}
+
+module "trigger_post_to_slack_client_error_alb" {
```

Comment:
It's only a notification when it goes into ALARM state. I think right now we do (we shouldn't be seeing any really) - we can dial it back when the API is under more load.